cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -136,14 +136,6 @@
   </property>
 
   <property>
-    <name>yarn.user</name>
```

Comment:
Might want to check if there are references in docs for this constant and remove those as well.